crates/livesplit-auto-splitting/src/process.rs

```diff
@@ -44,9 +44,17 @@ impl Process {
         process_list.refresh();
         let processes = process_list.processes_by_name(name);
 
-        // Use the process that was started the most recently, it's more
+        // Sorts the processes (asc) by numeric pid, to allow max_by_key to
+        // select the higher pid in case all records are equally maximum; otherwise
+        // use the process that was started the most recently, it's more
         // predictable for the user.
-        let pid = processes
+
+        let mut procs: Vec<&sysinfo::Process> = processes.map(|p| p).collect();
+
+        procs.sort_unstable_by_key(|p| p.pid().as_u32());
```

Comment:
Considering we only need the maximum here, we don't need to fully sort everything. Instead you can use a tuple in `max_by_key` made up of `(start_time, pid)`. That way it sorts primarily by `start_time` but falls back to the pid if they are equal.

Reply:
Thank you; A big light bulb went off as i was trying to do something similar.
Since Tuple implements `Ord` - it knows how to find the max value, correct?